<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Faculty Managing System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonBack">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>560</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 117, 117);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Student Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Request:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Professor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>210</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRequest">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>500</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudentNumber">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProfessor">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>500</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>NONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCamView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>pushButtonBack</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>labelName</zorder>
   <zorder>labelRequest</zorder>
   <zorder>labelStudentNumber</zorder>
   <zorder>labelProfessor</zorder>
   <zorder>labelCamView</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
